src/pip/_internal/commands/install.py

```diff
@@ -737,6 +737,7 @@ def reject_location_related_install_options(
     """If any location-changing --install-option arguments were passed for
     requirements or on the command-line, then show a deprecation warning.
     """
+    from pip._internal.utils.distutils_args import parse_distutils_args
```

Comment:
`parse_distutils_args` does not use distutils anymore, so it should be ok to keep this import global.

Reply:
Ah I was viewing an older `main`.